en/components/grids_templates/column_selection.md

```diff
@@ -0,0 +1,166 @@
+
+#### Keyboard combinations
+Basically there are three scenarios for the keyboard navigation of the **Column Selection**:
+- Single selection - <kbd>click</kbd> over the column cell if it is **selectable**.
+- Multi column selection - holding <kbd>ctrl</kbd> + <kbd>click</kbd> on every **selectable** header cell.
+- Range column selection - holding <kbd>shift</kbd> + <kbd>click</kbd> selects all **selectable** columns in between.
+
+#### API manipulations
+The **API** provides some additional capabilities when it comes to the **non-visible** columns such that, every **hidden** column could be marked as [`selected`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#selected) by setting the corresponding **setter**.
+
+> [!NOTE]
+> The above statement also applies to the [`IgxColumnGroupComponent`]({environment:angularApiUrl}/classes/igxcolumngroupcomponent.html), except that when the [`selected`]({environment:angularApiUrl}/classes/igxcolumngroupcomponent.html#selected) property is changed it respectively changes the state of it's descendants.
+
+More information regarding the API manipulations could be found in the [`API References`](#api-references) section.
+
+### Styling
+Before going deeper in the style manpulations, the core module and all component mixins need to be imported.
+```scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+```
+
+>[!NOTE]
+>Please note that [`row selection`](row_selection.md) and [`column selection`](column_selection.md) can't be manipulated   independently. They both are dependent on the same `variables`.
+
+With that being said, let's move on and change the **selection** and **hover** style.<br/>
+Following the simples approach let's define our custom **theme**.
+
+```scss
+$custom-grid-theme: igx-grid-theme(
+    $row-selected-background: steelblue,
+    $row-selected-text-color: gold,
+    $row-selected-hover-background: steelblue
+);
+```
+[`igx-grid-theme`]({environment:sassApiUrl}/index.html#function-igx-grid-theme) accepts several parameters but basically those are the three responsible for the respective styling:
+- **$row-selected-background** - sets the background of the selected fraction.
+- **$row-selected-text-color** - sets the text color of the selected fraction
+- **$row-selected-hover-background** - sets the color of the hovered cell or bunch of cells.
+
+#### Using CSS Variables
+The last step is to pass the custom `igx-grid` theme.
+
+```scss
+@include igx-css-vars($custom-grid-theme)
+```
+
+#### Using mixins
+In order to style components for Internet Explorer 11, we have to use different approach, since it doesn't support CSS variables.
+
+>[!NOTE]
+>If the component is using an [`Emulated`](../themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to `penetrate` this encapsulation using `::ng-deep`. In order to prevent the custom theme to leak to other components, be sure that you have included the `:host` selector before `::ng-deep`.
```

Comment:
```suggestion
>If the component is using the [`Emulated`](../themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to `penetrate` this encapsulation using `::ng-deep`. In order to prevent the custom theme from leaking into other components, be sure that you have included the `:host` selector before `::ng-deep`.
```